### PR TITLE
Cryptocurrency: Add interactive ui

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

Adds interactive UI to the cryptocurrency IA

## People to notify

@bsstoner 
@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/cryptocurrency

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Spice/Cryptocurrency.pm [hunk 1]
```diff
@@ -150,10 +140,14 @@ sub checkCurrencyCode {
...
             return;
         }
-        $endpoint = 'secondaries';
-        # API does not return the display name of the currency, so we must include it
-        $query = $normalized_number . $currDisplayName;
-        $query2 = $from;
+
+        if($to eq "") {
+            $to = "BTC";
+        }
```
Comment (bsstoner) [inline lib/DDG/Spice/Cryptocurrency.pm]: Not critical to launch with, but what do you think about defaulting to local currency instead like the other IA?

File: lib/DDG/Spice/Cryptocurrency.pm [hunk 2]
```diff
@@ -5,6 +5,8 @@ package DDG::Spice::Cryptocurrency;
 use DDG::Spice;
 with 'DDG::SpiceRole::NumberStyler';
 
+use Data::Dumper;
```
Comment (bsstoner) [inline lib/DDG/Spice/Cryptocurrency.pm]: not sure if you want to remove this from debugging, but not really doing any harm.

Comment (pjhampton) [inline lib/DDG/Spice/Cryptocurrency.pm]: Opps. My bad. Will remove today!

File: lib/DDG/Spice/Cryptocurrency.pm (hunk 1 above)
Comment (pjhampton) [inline lib/DDG/Spice/Cryptocurrency.pm]: Yeah, can do! Will submit a PR later

File: lib/DDG/Spice/Cryptocurrency.pm (hunk 2 above)
Comment (moollaza) [inline lib/DDG/Spice/Cryptocurrency.pm]: @pjhampton for what it's worth, `Data::Printer` is already added to all IA classes in the DDG module. You can simply use `p($foo)` to log structures 😄

**Pull request merged**